src/concat/requireify/parse-dependencies.js

```diff
@@ -0,0 +1,113 @@
+var _ = require('underscore');
+var path = require('path');
+var precinct = require('precinct');
+
+/**
+ * @param {Array<object>} files
+ */
+function parse(files) {

```

Comment:
This takes in the set of files being `require`-ified and finds all the `require` calls in each one. It then resolves any relative `require` calls to full paths. So, if `C:\foo\bar.js` has `require('./bat/view')`, this will resolve that to `C:\foo\bat\view.js`.